hedyorg/hedy
# 🖊️💻 Syntax highlighting for variables

Adds syntax highlighting for variables in both read only and main editors. It works by checking for variables definition in `is`,`ask`, etc and then highlighting the places where those names appear in the program. This is done in every update of the document, but since the size of our programs is small, this doesn't cause such a big overhead.

Fixes #4853 

**How to test**

Follow these steps to verify this PR works as intended:

* Go to the Hedy page and see that the variables are highlighted
